libmount/src/context_veritydev.c

```diff
@@ -186,10 +201,100 @@ int mnt_context_setup_veritydev(struct libmnt_context *cxt)
 		rc = -EINVAL;
 	}
 
+	if (rc == 0) {
+		int dl_flags = RTLD_LAZY;
```

Comment:
`RTLD_LOCAL` is the default anyway, but explicitly using `RTLD_LAZY | RTLD_LOCAL` is probably clearer.

Reply:
Sure, done